docs/ingestion/native-batch.md

```diff
@@ -846,6 +847,15 @@ S3 Object:
 |bucket|Name of the S3 bucket|None|yes|
 |path|The path where data is located.|None|yes|
 
+Properties Object:
+
+|property|description|default|required?|
+|--------|-----------|-------|---------|
+|accessKeyId|S3 access key for this S3 InputSource|None|yes if secretAccessKey is given|
```

Comment:
I think these can just be shortened to `S3 access key` and `S3 secret key`

Reply:
What do you mean? The description or the property itself?

Reply:
Oh i see. Wanted to emphasize that these configs will only be applied to this input source (not the deep storage, etc.)